controllers/topics-controller.js

```diff
@@ -0,0 +1,21 @@
+
+const {fetchAllTopics} = require ('../models/topics-models')
+
+
+
+
+
+exports.getAllTopics = (req, res, next) => {
+
+    fetchAllTopics()
+    .then((topics)=>{
+
+        if (topics.length === 0) {
```

Comment:
You wouldn't throw errors in the then block of the controller. That is the job of the model. If we are in this part of the controller it is assumed that everything is ok and the information sent back from the model is good. 
Is it likely that the database wouldn't have any topics in the table?